src/linker/utilities/data_utils.py

```diff
@@ -51,3 +51,11 @@ def validate_dummy_output(filepath: Path) -> None:
         raise LookupError(
             f"Data file {filepath} is missing required column(s) {missing_columns}"
         )
+
+
+def validate_dummy_input(filepath: Path) -> Optional[List[str]]:
+    "Wrap the output file validator for now, since it is the same"
+    try:
+        validate_dummy_output(filepath)
+    except Exception as e:
+        return [e.args[0]]
```

Comment:
What does this part do?

Reply:
It's just to make it fit with the logging format of the input validation, which is different from the output validation. The input validation is batched with the other config validations, so we need to return the log message specifically instead of raising an exception directly, while the output validations are free to just raise an error on their own.

Reply:
Makes sense! I'd copy and paste that into a comment :)